h2/src/docsrc/html/features.html

```diff
@@ -1826,6 +1828,80 @@ <h2 id="cache_settings">Cache Settings</h2>
 call <code>SELECT * FROM INFORMATION_SCHEMA.SETTINGS</code>. The number of pages read / written
 is listed.
 </p>
+<h2 id="external_authentication">External authentication</h2>
+<p>
+External authentication allows to optionally validate user credentials externally (JAAS,LDAP,custom classes).
+Is also possible to temporary assign roles to to externally authenticated users
+
+</p>
+<p>Master user cannot be externally authenticated</p>
+<p>
+To enable external authentication on a database execute statement <code>SET AUTHENTICATOR='on'</code>. This setting in persisted on the database. 
```

Comment:
I think you mean
    SET AUTHENTICATOR ON

Reply:
To don't change set syntax I will use a boolean value SET AUTHENTICATOR TRUE|FALSE